src/dbnode/network/server/tchannelthrift/node/service_test.go

```diff
@@ -491,6 +582,55 @@ func TestServiceFetchBatchRaw(t *testing.T) {
 	}
 }
 
+func TestServiceFetchBatchRawUnknownError(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockDB := storage.NewMockDatabase(ctrl)
+	mockDB.EXPECT().Options().Return(testStorageOpts).AnyTimes()
+	mockDB.EXPECT().IsOverloaded().Return(false)
+
+	service := NewService(mockDB, testTChannelThriftOptions).(*service)
+	tctx, _ := tchannelthrift.NewContext(time.Minute)
+	ctx := tchannelthrift.Context(tctx)
+	defer ctx.Close()
+
+	start := time.Now().Add(-2 * time.Hour)
+	end := start.Add(2 * time.Hour)
+
+	start, end = start.Truncate(time.Second), end.Truncate(time.Second)
+
+	nsID := "metrics"
+	unknownErr := fmt.Errorf("unknown-err")
+
+	series := map[string][]struct {
```

Comment:
Can you just do `series := map[string]ts.Datapoint`?

Reply:
yeah def could, just following the pattern in this file to keep the tests uniform.